lib/template.html

```diff
@@ -65,6 +65,10 @@
   {% assign seo_author_twitter = seo_author_twitter | replace:"@","" %}
 {% endif %}
 
+{% if page.date_modified or page.last_modified_at or page.date %}
+  {% assign seo_date_modified = page.seo.date_modified | default: page.last_modified_at | default: page.date %}
```

Comment:
Addition: When I check out websites that already use JSON-LD, I note that they use dateModified even if it is the same as the publish date. 

> view-source:http://www.theverge.com/2017/4/3/15123170/bmw-520i-2017-5-series-review